wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,317 @@
+    path: path.resolve(__dirname, "dist"),
+    filename: "index.js",
+  },
+  mode: "development",
+};
+```
+
+Then, edit `package.json` to include a build script:
+
+```json title="package.json"
+"scripts": {
+  "build": "webpack"
+}
+```
+
+When you run `npm run build` from your terminal, Webpack bundles the content in `src/index.js` into `dist/main.js`, which is linked in `index.html`.
+
+### 3. Detect MetaMask
+
+:::caution
+
+Using the `window.ethereum` provider object to detect MetaMask is deprecated. We recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md) for production environments.
+
+:::
+
+Install the `@metamask/detect-provider` module in your project directory:
+
+```bash
+npm i @metamask/detect-provider
+```
+
+Add the following code to `src/index.js` to detect the MetaMask provider using `@metamask/detect-provider`:
+
+```js title="index.js"
+import detectEthereumProvider from "@metamask/detect-provider";
+
+async function setup() {
+  const provider = await detectEthereumProvider();
+
+  if (provider && provider === window.ethereum) {
+    console.log("MetaMask is available!");
+    startApp(provider); // Initialize your dapp with MetaMask.
+  } else {
+    console.log("Please install MetaMask!");
+  }
+}
+
+function startApp(provider) {
+  if (provider !== window.ethereum) {
+    console.error("Do you have multiple wallets installed?");
+  }
+}
+
+window.addEventListener("load", setup);
+```
+
+#### Serve the project
+
+To test and verify your project works with MetaMask, serve your project through a local server. 
+If using VS Code and [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), right-click the `index.html` file, and select **Open with Live Server**. 
+ 
+Alternatively, you can install `http-server` using the following command:
+
+```bash
+npm install -g http-server
+```
+
+Then navigate to the project directory, and run `http-server`. 
+
+Open your browser and go to the provided local server URL (typically http://127.0.0.1:8080).
+
+### 4. Determine a user's connected network
+
+[Detect the user's network](#3-detect-metamask) to ensure all RPC requests are submitted to the currently connected network.
+Add the following code to `src/index.js`, which uses the [`eth_chainId`](/wallet/reference/eth_chainId)
+RPC method to detect the chain ID of the user's current network, and listens to the [`chainChanged`](/wallet/reference/provider-api/#chainchanged) provider event to detect when the user changes networks:
+
+```jsx title="index.js"
+const chainId = await window.ethereum 
+    .request({ method: "eth_chainId" });
+
+window.ethereum
+    .on("chainChanged", handleChainChanged);
+
+function handleChainChanged(chainId) {
+    // We recommend reloading the page, unless you must do otherwise.
+    window.location.reload();
+}
+```
+For more information on Chain IDs, see [Detect a user's network](../tutorials/javascript-dapp-simple.md)
+
+### 5. Access an account
```

Comment:
```suggestion
### 6. Access a user's accounts
```

Reply:
I think I renamed this, because there's duplicate naming for some sections and pages. We have a page titled access "Access a user's accounts" and then this section. I think that could be a bit confusing since we are not reusing content and it differs. I know that the actual page link is different, but for someone navigating the docs or searching could this be a bit confusing?

Reply:
I think it's fine for it to be the same, it's essentially the same content and it's a very simple task. This should be changed to "accounts" anyway because `eth_requestAccounts` can return multiple accounts.